pom.xml

```diff
@@ -511,40 +511,5 @@
 			</plugins>
 		    </build>
 		</profile>
-		<profile>
```

Comment:
We will still need this profile. Please do not kill it. We will need to keep there all optimization for faster building under docker - for example "not building" particular modules needs to be moved here as explicit listing of modules which we have to build.